tools/crmadmin.c

```diff
@@ -244,7 +244,8 @@ main(int argc, char **argv)
                                          NULL);
             break;
         case cmd_list_nodes:
-            rc = pcmk__list_nodes(out, options.optarg, options.bash_export);
+            rc = pcmk__list_nodes(out, options.optarg,
+                                  (options.bash_export? true : false));
```

Comment:
I don't think putting this in a ternary makes it any more type-correct. The first part (`options.bash_export?`) just checks whether the value is truth-y and returns true if so, otherwise false. Which is the same thing that would happen if we passed the `gboolean` to `pcmk__list_nodes()` and let it be cast implicitly to `bool`.

Reply:
true (no pun intended)